lib/serializer/ESerializationVersion.h

```diff
@@ -42,8 +42,9 @@ enum class ESerializationVersion : int32_t
 	REWARDABLE_EXTENSIONS, // new functionality for rewardable objects
 	FLAGGABLE_BONUS_SYSTEM_NODE, // flaggable objects now contain bonus system node
 	RANDOMIZATION_REWORK, // random rolls logic has been moved to server
+  COLOR_FILTER, // color filter cheat
```

Comment:
1. we only maintain serialization compatibility for saves, not for netpacks. Multiplayer between 1.X and 1.Y is not possible. Small exception that I usually try to avoid is hotfix MP compatibility,  e.g 1.X.Y and 1.X.Z should be generally compatible. But this is not the case here.
2. Spaces instead of tabs?